src/datatype/functions_svecf32.rs

```diff
@@ -43,3 +540,163 @@ fn _vectors_to_svector(
     }
     SVecf32Output::new(SVecf32Borrowed::new(dims.get(), &indexes, &values))
 }
+
+/// divide a sparse vector by a scalar.
+#[pgrx::pg_extern(immutable, strict, parallel_safe)]
+fn _vectors_svecf32_div(vector: SVecf32Input<'_>, scalar: f32) -> SVecf32Output {
```

Comment:
1. where is it used?
2. if scalar is very large, and values in vector are small, `values` contains zeros so `SVecf32Output::new` panics. You can add a function to receive input containing zeros as a helper.

Reply:
> 1. where is it used?

It was used in old implementation. It is useless now. 

>     2. if scalar is very large, and values in vector are small, `values` contains zeros so `SVecf32Output::new` panics. You can add a function to receive input containing zeros as a helper.

Good catch! Thanks.